hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -121,6 +131,21 @@ def _on_retire(self,
 
         return changes
 
+    def _on_wipe(self, verbose: bool) -> List[Trace]:
+        '''This is run on wipe cycles'''
+        if self.state.wiping_cnt == 1:
+            changes = []
+        else:
+            changes = self.state.changes()
```

Comment:
This looks a little magical to me! What's special about the first cycle of the wipe? It looks like we're suppressing the reporting of some change: maybe that logic belongs somewhere else?